vgo/src/main/kotlin/com/jzbrooks/vgo/Application.kt

```diff
@@ -266,12 +266,12 @@ class Application {
 > vgo [options] [file/directory]
 
 Options:
-  -h --help       print this message
-  -o --output     file or directory, if not provided the input will be overwritten
-  -s --stats      print statistics on processed files to standard out
-  -v --version    print the version number
-  --indent value  write files with value columns of indentation
-  --format value  output format (svg, vd, etc)
+  -h --help          print this message
+  -o --output        file or directory, if not provided the input will be overwritten
+  -s --stats         print statistics on processed files to standard out
+  -v --version       print the version number
+  --indent value     write files with value columns of indentation
+  -f --format value  output format (svg, vd, etc)
```

Comment:
Is this solving a problem you have?

My intention behind this not having a shorthand was clarity over brevity for options with an associated value.

Reply:
Nope. Just want an easier usage, or it could be `-F`.